src/main/java/se/sundsvall/notes/api/NotesResource.java

```diff
@@ -109,4 +111,18 @@ public ResponseEntity<Void> deleteNoteById(
 		noteService.deleteNoteById(id);
 		return noContent().build();
 	}
+
+	@GetMapping(path = "/{id}/difference", produces = { APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE })
+	@Operation(summary = "Get change-events by ID, fromRevision and toRevision")
+	@ApiResponse(responseCode = "200", description = "Successful operation", content = @Content(mediaType = APPLICATION_JSON_VALUE, schema = @Schema(implementation = Note.class)))
+	@ApiResponse(responseCode = "400", description = "Bad request", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(oneOf = { Problem.class, ConstraintViolationProblem.class })))
+	@ApiResponse(responseCode = "404", description = "Not found", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
+	@ApiResponse(responseCode = "500", description = "Internal Server error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
+	public ResponseEntity<DifferenceResponse> getDifferenceByVersions(
+		@Parameter(name = "id", description = "Note ID", example = "b82bd8ac-1507-4d9a-958d-369261eecc15") @ValidUuid @PathVariable final String id,
+		@Parameter(name = "from", description = "From version", example = "1") @RequestParam(required = false) final Integer from,
```

Comment:
Maybe add a constraint that prevents negative numbers?
Maybe something like:

`@Range(min = 0, max = Integer.MAX_VALUE)`